apps/backend/src/modules/rating/controller.ts

```diff
@@ -0,0 +1,334 @@
+import { RatingModel } from '@repo/common';
+import {
+  ClassIdentifier,
+  MetricName,
+  RatingIdentifier
+} from "../../generated-types/graphql";
+import {
+  formatUserRatings,
+  formatAggregatedRatings
+} from "./formatter";
+
+// TODO: get list of all avaiable semesters with ratings (check for class offered?)
+// TODO: get list of all available semesters class offered in
+// TODO: get user ratings for given class
+
+export const getClassRatingsForUser = async (
+  context: any,
+  classIdentifier: ClassIdentifier
+) => {
+  if (!context.user._id) throw new Error("Unauthorized");
+
+  const aggregated = await ratingAggregator(classIdentifier);
+  if (!aggregated.length) return null;
+
+  return formatAggregatedRatings(aggregated[0]);
+};
+
+export const createRating = async (
+  context: any, 
+  ratingIdentifier: RatingIdentifier,
+  email: string,
```

Comment:
You don't need to take in `email` because you can get the user from the `getUser(context)` function in the `user` module.